### DISCUSSION TITLE
Display HODL value

### DISCUSSION THREAD

**Discussion by t3ch9** [vsaveris/trading-technical-indicators#34] [Ideas]
Display portfolio value if only bought at start date and not sold to be used as a benchmark to compare with results of trading signal simulation.

**Comment by vsaveris** [comment]
Hi,

can you please add some more details about your idea?

**Reply by t3ch9** [reply]
Calculate the Buy & Hold value. This would serve as benchmark to demonstrate the effectiveness of indicator strategy vs buy & hold strategy.
